src/interceptors/ClientRequest/NodeClientRequest.test.ts

```diff
@@ -151,6 +148,10 @@ test('emits the ENOTFOUND error connecting to a non-existing hostname given no m
     done()
   })
 
+  request.on('response', () => {
```

Comment:
Should we remove this if the issue has been resolved?
The response callback shouldn't even be called in this scenario, as the request errors on trying to connect to a non-existing endpoint.

Reply:
Removed